### ISSUE TITLE
Write a comprehensive page on the process of making an MRP

### ISSUE THREAD

**Issue by AThousandShips** [godotengine/godot-docs#9678]
**Your Godot version:**

N/A

**Issue description:**

Making an MRP is a complicated and difficult process even for experienced contributors, and after discussion in rocket-chat and general experience with triaging I believe we should create a page with instructions on how to effectively make a good MRP, with things like:
* Tips on how to identify the minimal state if you are starting from an existing project
* How to create a good MRP from scratch with ideas for how to showcase the behaviour effectively, such as:
  - ideas for adding interactivity to trigger behaviour
  - ideas for adding configurability to make it easier to display different cases
  - good sources for free and permissively licensed material to put into your project, such as audio, art, meshes, etc., this has often been a struggle for me working with making MRPs

Further ideas are welcome on what to put into this

**URL to the documentation page (if already existing):**

**Comment by afranke** [comment]
Spelling out what MRP stands for would be a good start. It currently is mentioned on two pages and none of them have the acronym expanded. I’m guessing it is a minimum reproducer project, but it isn’t totally clear.

**Comment by AThousandShips** [comment]
It does specify [here](https://contributing.godotengine.org/en/latest/triage/workflow.html#testing-an-issue) but it is not entirely clear, however this is being worked on currently (It stands for "minimal reproduction project", which is clarified in the issue report template)

**Comment by afranke** [comment]
> It does specify [here](https://contributing.godotengine.org/en/latest/triage/workflow.html#testing-an-issue)

Right, there’s an explanation of what it is, but no spelling out of the acronym, as “minimal project that reproduces” doesn’t exactly give us M.R.P. 🙂 

For additional context, the two pages I was referring to are [Introduction to issue triage](https://contributing.godotengine.org/en/latest/triage/about.html) and [Bug triage overview](https://contributing.godotengine.org/en/latest/triage/workflow.html). The acronym should be expanded on these two page the first time it is used.

> this is being worked on currently

Awesome. 🚀
